dev/AppLifecycle/LaunchActivatedEventArgs.h

```diff
@@ -9,21 +9,21 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
 {
     using namespace winrt::Windows::ApplicationModel::Activation;
 
-    static PCWSTR c_launchContractId = L"Windows.Launch";
+    constexpr PCWSTR c_launchContractId = L"Windows.Launch";
 
-    class LaunchActivatedEventArgs : public winrt::implements<LaunchActivatedEventArgs,
-        ActivatedEventArgsBase, ILaunchActivatedEventArgs, IInternalValueMarshalable>
+    class LaunchActivatedEventArgs : public winrt::implements<LaunchActivatedEventArgs, ILaunchActivatedEventArgs,
+        ActivatedEventArgsBase, IInternalValueMarshalable>
     {
     public:
-        LaunchActivatedEventArgs(const std::wstring& args) : m_args(args)
+        LaunchActivatedEventArgs(const std::wstring& args) : m_args(std::move(args))
```

Comment:
`std::move` from a `const` variable has no effect.

Reply:
I've opened a bug for @aeloros to address the AppLifecycle comments on a separate PR.: https://github.com/microsoft/ProjectReunion/issues/926

FYI @loneursid